### PR TITLE
ODM-11596: remove obsolete e2e/parsing tests

### PR DESCRIPTION
**Pull request by m-smazhevsky**
These tests seems to be obsolete: some of them rely on some launched server, another ones are checking default parameters. Anyway they doesn't look useful. Still can fix and preserve some of them if you like.

### CODE REVIEW

**Review by genestack-okunitsyn — DISMISSED** [review]

File: test/test_arguments.py [hunk 1]
```diff
@@ -1,96 +0,0 @@
...
-def test_token_without_username():
-    parser = make_connection_parser()
-    some_token = 'some_token'
-    args = parser.parse_args(['--token', some_token])
-    user = get_user(args)
-    assert user.token == some_token
-
-
-if __name__ == '__main__':
-    pytest.main(['-v', '--tb', 'short', __file__])
```
Comment (viapivov) [inline test/test_arguments.py:96]: I guess we can put a little effort in saving these tests, WDYT?

EDIT: tests of cli parser functionality, hell with others

Comment (m-smazhevsky) [reply test/test_arguments.py:96]: Ok, I saved some, hope I guessed the ones you were thinking about.

**Review by viapivov — DISMISSED** [review]
Yep, thanks!

**Review by genestack-okunitsyn — DISMISSED** [review]

**Review by viapivov — APPROVED** [review]

**Review by genestack-okunitsyn — APPROVED** [review]

File: test/test_arguments.py (hunk 1 above)
Comment (m-smazhevsky) [reply test/test_arguments.py:96]: resolving this

**Pull request merged**